contracts/tests/sbtc-withdrawal.test.ts

```diff
@@ -233,14 +233,40 @@ describe("initiating a withdrawal request", () => {
   });
 });
 
+test("max-fee must be accounted for", () => {
+  txOk(
+    deposit.completeDepositWrapper({
+      txid: new Uint8Array(32).fill(0),
+      voutIndex: 0,
+      amount: 4000n,
+      recipient: alice,
+    }),
+    deployer
+  );
+  const receipt = txErr(
+    withdrawal.initiateWithdrawalRequest({
+      amount: 4000n,
+      recipient: alicePoxAddr,
+      maxFee: 1000000n,
```

Comment:
I would probably just use `maxFee: 1n` here to avoid the reader (i.e. me) wondering "is there 999999 sats somewhere i'm not seeing?" :sweat_smile:

Reply:
Good idea, will do.

Reply:
Updated in https://github.com/stacks-network/sbtc/pull/498/commits/5080d28416a188b29d69edc2df18165b07f99494.